openjdk-testing/src/main/java/org/conscrypt/testing/TestClient.java

```diff
@@ -39,8 +37,8 @@ public TestClient(SSLSocket socket) {
     public void start() {
         try {
             socket.startHandshake();
-            input = new BufferedInputStream(socket.getInputStream());
-            output = new BufferedOutputStream(socket.getOutputStream());
+            input = socket.getInputStream();
```

Comment:
I don't think any realistic application would read 64 bytes from the socket at a time.

Message size represents the chunk size that we're passing through the SSL engine at a time.

The buffer size is just how much data we're reading from the socket ... and if it's too small, socket reads will dominate and the benchmark will be a wash. For writes, we're just sending the message, itself.

Perhaps the solution here is to use Buffered I/O if the messageSize < 8KiB?